presubmit-service/check-pr.ts

```diff
@@ -0,0 +1,12 @@
+import * as shell from 'shelljs';
+
+/** Determine if the PR can apply to a repo's targeted branch. */
+export const checkCherryPickibilityOfPr = (org: string, repo: string, pr: string, branch: string) => {
+  const prCheck = shell.exec(`./check_if_pr_can_cherry_pick.sh ${org} ${repo} ${pr} ${branch}`);
+  if (prCheck.code === 0) {
+    return {status: 'success', message: ''};
+  } else {
+    // TODO(josephperrott): Determine what information should be provided with failure outputs.
+    return {status: 'failure', message: ''};
```

Comment:
It might make sense to have a function that just returns a boolean for whether the PR can cherry-pick cleanly, separate from whatever message is delivered for that

Reply:
Added comment as discussed.